docs/api/attachments.md

```diff
@@ -0,0 +1,176 @@
+# Attachments
+Supports adding, viewing, deleting, and updating attachments.
+
+## Description
+Attachments are used to support any kind of external file. These are often PDFs or images. Attachments are used for Material Transfer Agreements, Institutional Review Board documents, and sequence maps for materials among other things.
+
+## Add a new attachment:
+
+**Request**:
+
+`POST` `/attachments/`
+
+Parameters:
+
+Name            | Type   | Description
+----------------|--------|---
+filename        | string | The filename of the attachment.
+description     | string | A description of the attachment.
+s3_bucket       | string | The url of the s3_bucket where your attachment file is stored.
+s3_key          | string | The key of your s3 object.
+sequence_map_for| json   | A material object in json. The material which this attachment is a sequence map for.
+
+*Note:*
+
+- `sequence_map_for` is optional, and should only be provided when the attachment is a sequence map.
+- **[Authorization Protected](authentication.md)**
+
+**Response**:
+
+```json
+Content-Type application/json
+201 CREATED
+
+{
+   "id":18,
+   "filename":"attachment_file",
+   "description":"A file for testing",
+   "s3_bucket":"https://bucket-name.s3.region.amazonaws.com/keyname",
+   "s3_key":"s3 key",
+   "deleted":"None",
```

Comment:
Is this the safedelete field or `s3_resource_deleted`? I think it maybe is the former but is intended to be the latter?

Reply:
Good catch. I just changed the serializer.